.github/workflows/check-e2e.yml

```diff
@@ -58,10 +61,25 @@ jobs:
         if: steps.playwright-dep-cache.outputs.cache-hit == 'true'
         run: npx playwright install-deps
 
+      - id: auth
+        uses: google-github-actions/auth@v1
+        with:
+          workload_identity_provider: "projects/281042598092/locations/global/workloadIdentityPools/github-actions/providers/google-cloud"
+          service_account: "github-actions@radicle-services.iam.gserviceaccount.com"
+
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v1
+
+      - name: Download radicle-httpd executable
+        run: |
+          gsutil cp gs://heartwood-artifacts/1f55d7a32750b3e63c56aad00370411b90420a29/radicle-httpd ./
+          chmod a+x ./radicle-httpd
```

Comment:
Could we actually move this bit into `scripts/run-httpd-with-fixtures`? So that we specify the `rev` in a single place?

Also would prefer if the binary ended up in `tests/tmp` rather than the repo root.